src/main/java/org/terasology/tutorialWorldGeneration/HouseProvider.java

```diff
@@ -15,17 +15,17 @@
  */
 package org.terasology.tutorialWorldGeneration;
 
-import org.terasology.math.Region3i;
-import org.terasology.utilities.procedural.Noise;
-import org.terasology.utilities.procedural.WhiteNoise;
-import org.terasology.world.generation.Border3D;
-import org.terasology.world.generation.Facet;
-import org.terasology.world.generation.FacetBorder;
-import org.terasology.world.generation.FacetProvider;
-import org.terasology.world.generation.GeneratingRegion;
-import org.terasology.world.generation.Produces;
-import org.terasology.world.generation.Requires;
-import org.terasology.world.generation.facets.SurfacesFacet;
+import org.terasology.engine.world.block.BlockRegion;
+import org.terasology.engine.utilities.procedural.Noise;
+import org.terasology.engine.utilities.procedural.WhiteNoise;
+import org.terasology.engine.world.generation.Border3D;
```

Comment:
These imports are not ordered correctly.
IntelliJ has a nice feature to "Optimize Imports" so that no unused imports are left and they are organized according to code style rules (https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#optimize-imports)